org.gnome.Yelp.desktop.in.in

```diff
@@ -1,10 +1,10 @@
 [Desktop Entry]
-_Name=Help
-_Comment=Get help with GNOME
-_Keywords=documentation;information;manual;help;
+Name=Help Center
+Comment=Get help with GNOME
+Keywords=documentation;information;manual;help;
 OnlyShowIn=GNOME;Unity;
 Exec=yelp %u
-Icon=help-browser
+Icon=eos-app-org.gnome.Yelp
```

Comment:
```suggestion
Icon=help-browser
```

https://github.com/endlessm/eos-shell-content/blob/d04b738c8ca6a1f92466439b054184fbab113446/bundle/desktops/org.gnome.Yelp.desktop#L33 changes the icon.

Reply:
Ah right. I repeatedly miss to look for eos-shell-content thats plays a role here.

Addressed.